File: src/main/java/arc/arc/hooks/zauction/AuctionItemDto.java
package arc.arc.hooks.zauction;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;
import java.util.List;

@Data @AllArgsConstructor
@NoArgsConstructor
@Builder
public class AuctionItemDto {

    String display;
    String seller;
    String price;
    long expire;
    String category;
    int amount;
    int priority;
    String uuid;
    boolean exist;
    List<String> lore = new ArrayList<>();

}

File: src/main/kotlin/ru/arc/hooks/zauction/AuctionItemDto.kt
package ru.arc.hooks.zauction

data class AuctionItemDto(
    val display: String? = null,
    val seller: String? = null,
    val price: String? = null,
    val expire: Long = 0L,
    val category: String? = null,
    val amount: Int = 0,
    val priority: Int = 0,
    val uuid: String? = null,
    val exist: Boolean = false,
    val lore: List<String> = emptyList(),
)
